### PR TITLE
Adding html type allowing a pure html as an option

### PR DESCRIPTION
**Pull request by drmikecrowe**
This change allows creating an option as follows:

https://github.com/drmikecrowe/mbfcext/blob/master/MBFCext/extension/options.js#L7

### CODE REVIEW

File: chrome-options.js [hunk 1]
```diff
@@ -237,6 +237,10 @@
     return !hashOption && $('<h3>').text(option.desc);
   }
 
+  function addHtml(option) {
+    console.log(option);
```
Comment (fent) [inline chrome-options.js]: Should be deleted

File: chrome-options.js [hunk 2]
```diff
@@ -246,6 +250,10 @@
       hashPosition++;
     }
 
+    if (option.type == 'html') {
+      return addHtml(option);
+    }
```
Comment (fent) [inline chrome-options.js]: I'd rather put this next to the [layout options](https://github.com/fent/chrome-options#layout-fields)

Either after `case 'row':` below, or after `case 'h3':` in `addTabOptions()`

**Comment by drmikecrowe** [comment]
I'd appreciate your help in this.  Seems like I tried putting it after the
h3, but I couldn't get it to work.

**Comment by fent** [comment]
Thinking more about it, I think it should be placed next to `column` and `row`

```js
      case 'html':
        $option = addHtml(option);
        break;
```

Although, it should be like the other layout types and be placed in `chrome.options.base.html`

**Comment by fent** [comment]
Merged with 0965e21a62b274e86c74531d0d6e6a769f4d818c. Thanks again!

**Pull request closed without merging**